.github/workflows/test-extension.yml

```diff
@@ -0,0 +1,23 @@
+name: Test Extension
+on:
+  push:
+    branches:
+      - master
+jobs:
+  test-extension:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
```

Comment:
Running on an array of versions (from minimum supported till like 3.8) is probably a good idea.